drivers/ethernet/eth_numaker_priv.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2023 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_ETHERNET_ETH_NUMAKER_PRIV_H_
+#define ZEPHYR_DRIVERS_ETHERNET_ETH_NUMAKER_PRIV_H_
+
+#include <zephyr/types.h>
+
+#define NU_ETH_MAX_FLEN             (1518)
+
+#define NU_HWADDR_SIZE              (6)
```

Comment:
You had hardcoded value 6 used in .c file for mac length, perhaps those could be replaced by this define?

Reply:
Thanks of your advice.
Updated at commit  44c44651b

Reply:
Hi @jukkar,
Appreciate your comment.
Finished the modification based on your above 7 advice.